monkey/infection_monkey/Pipfile

```diff
@@ -4,10 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-altgraph = "*"
```

Comment:
This is required, otherwise our fork of pyinstaller isn't properly installed and agents fail to build. We should probably add a comment to that effect.